app/views/services/edit.html.erb

```diff
@@ -105,6 +107,7 @@
       <li><a href="#add-page" data-page-type="checkanswers">Check answers page</a></li>
       <li><a href="#add-page" data-page-type="confirmation">Confirmation page</a></li>
       <li><a href="#add-page" data-page-type="content">Content page</a></li>
+      <li><a href="#add-page" data-page-type="exit">Exit page</a></li>
```

Comment:
sorry one more thing that i forgot. exit pages are going to be released at the same time as branching. so we need to only allow them to be added if `ENV['BRANCHING']` is set to `enabled`.